src/hdcontroller.ts

```diff
@@ -238,12 +245,13 @@ export class HDProxydClient {
      * creation and its sending.
      */
     private _handleRequest(method: string, stream: Stream|null,
-                           size: number, key: string, log: any,
-                           callback: HDProxydCallback, params: any,
-                           payload: any): void {
-        const headers = params.headers ? params.headers : {};
+        size: number, key: string, log: werelogs.RequestLogger,
+        callback: HDProxydCallback, params: Params,
+        payload: object | undefined): void {
+        // tslint:disable-next-line:no-any
```

Comment:
why the annotation, you explciitely typed the next line...

Reply:
'content-length'?: number; [key: string]: any is causing an error thus the reason behind the annotation